### DISCUSSION TITLE
filenameUTF8 always false

### DISCUSSION THREAD

**Discussion by nenge123** [gildas-lormeau/zip.js#395] [Q&A]
byte utf8(single byte)/gbk rawBitFlag is 0
byte utf8(more byte) rawBitFlag is 9
btye gbk rawBitFlag is 8

**Comment by gildas-lormeau** [comment]
When reading a zip file, this boolean is set to `true` if the bit 11 of the general purpose bit flag is set, see 4.4.4 here: https://pkware.cachefly.net/webdocs/APPNOTE/APPNOTE-6.3.9.TXT.
When writing a zip file, this boolean is always set to `true` because zip.js always uses UTF-8 to encode filenames.
